### PR TITLE
[IMP] MRP: Scrap during manufacturing

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR almost fully rewrites the "Scrap during manufacturing" doc to replace the tablet view workflow with the Shop Floor workflow, as well as to make general improvements to the doc.

### CODE REVIEW

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 1]
```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
...
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Open the Scrap pop-up window
+============================
 
-.. image:: scrap_manufacturing/scrap-button.png
-   :align: center
-   :alt: The scrap button on a manufacturing order.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window. Once
+opened, the pop-up window can be filled out in the same way, regardless of how it was accessed.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:36]: > the pop-up window can be filled out in the same way, regardless of how it was accessed.

I'm a little confused about this sentence here. The pop-up window is filled in the same way as...?

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 2]
```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
...
+However, the method for accessing the window differs depending on the type of scrapping taking
+place.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:38]: Can you help me elaborate what you meant by the different kind of scrapping? Are you referring to the different subsections below?

In that case, how about rewording this sentence to something like:

```suggestion
The
following are three ways of navigating to the :guilabel:`Scrap` window, depending on the app and
product type.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 3]
```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
...
-Finally, click :guilabel:`Done` to scrap the component.
+Scrap component from Manufacturing
+----------------------------------
 
-.. image:: scrap_manufacturing/scrap-pop-up.png
-   :align: center
-   :alt: The Scrap pop-up window.
+To scrap a component from an |MO|, begin by navigating to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, and then select an |MO|. At the top of the |MO|, click the
+:guilabel:`Scrap` button to open the :guilabel:`Scrap` pop-up window.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:45]:
```suggestion
Operations --> Manufacturing Orders`, and then select an |MO|.

At the top of the |MO|, click the :guilabel:`Scrap` button to open the :guilabel:`Scrap` pop-up
window.
```

Totally optional, but since this section is a little short, maybe you can include an image of where the scrap button is. I heard from some ppl in support that they like seeing more pictures, as well! If you decide to do this, I recommend adding screenshots to all the subsections (:

image

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 4]
```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
...
+<three_step_manufacturing>` manufacturing enabled, since components are not picked as part of the
+:doc:`one-step <one_step_manufacturing>` manufacturing process.
 
-Since the components have been consumed to create the finished product, they will no longer appear
-in the :guilabel:`Product` drop-down menu. Instead, the finished product will be available as an
-option. Select the finished product and enter the quantity to be scrapped in the
-:guilabel:`Quantity` field. Click :guilabel:`Done` to scrap the finished product.
+.. image:: scrap_manufacturing/scrap-window.png
+  :align: center
+  :alt: The Scrap pop-up window.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:93]: Slightly not aligned!
```suggestion
   :align: center
   :alt: The Scrap pop-up window.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 5]
```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
...
+After opening the scrap pop-up window using one of the methods :ref:`detailed above
+<scrap_components/scrap-window>`, select the component or finished product being scrapped, from the
+:guilabel:`Product` drop-down menu. In the :guilabel:`Quantity` field, enter the quantity being
+scrapped.
 
-Scrap finished products
-=======================
+By default, the :guilabel:`Source Location` field is set to the warehouse's pre-production location,
+while the :guilabel:`Scrap Location` field is set to the :guilabel:`Virtual Locations/Scrap`
+location. If either the source or scrap location should be changed, select a different location from
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:82]: Can also maybe elaborate here again that the "Scrap Location" field designates where the scrapped products go

**Review by Felicious — APPROVED** [review]
Great job on this doc, @hojo-odoo !

I just had some minor comments-- all of the flows are accurate and I like how you split navigating to the scrap window and filling out the scrap form in two sections!

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 3 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:45]: I try not to include pictures that only point out a single button, especially when it's very easy to find.

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 5 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:82]: I feel like that would be a bit redundant.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 6]
```diff
@@ -11,7 +11,7 @@ Warehouse
 In Odoo, a **Warehouse** is the actual building/place in which a company's items are stocked.
 Multiple warehouses can be set up in Odoo and the user can create moves between warehouses.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:12]:
```suggestion
In Odoo, a *warehouse* is the actual building/place, in which a company's items are stocked and
stored. Multiple warehouses can be set up in Odoo, and the user can create moves between warehouses.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 7]
```diff
@@ -11,9 +13,13 @@ it in a virtual location titled *Virtual Locations/Scrap*. A virtual location is
...
-Odoo *Manufacturing* allows for both components and finished products to be scrapped within a
-manufacturing order. The specific type of item that can be scrapped during a manufacturing order
-depends on the stage of the manufacturing process.
+.. seealso::
+   For more information, see the documentation about the different types of :ref:`locations
+   <inventory/warehouses_storage/inventory_management/warehouses-locations/locations>`
+
+Components can be scrapped from both the *Manufacturing* app and the *Shop Floor* module, before the
+associated manufacturing order (MO) is closed. Finished products can be only be scrapped from the
+*Manufacturing* app, and only after closing the associated |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:22]:
```suggestion
associated manufacturing order (MO) is closed. Finished products can only be scrapped from the
*Manufacturing* app, and only after closing the associated |MO|.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 8]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:31]: MENG has recently informed me that we should use the simple "search bar" rather than a GUI label for the search bar, after she saw I was using this very same format. Either way, I think it would be wise to broach this subject with JERO (and/or ZST) to find an agreed-upon solution, because I frequently include the search bar in my docs, and would like to avoid any discrepancies regarding this.

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 9]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
...
-Scrap manufacturing components
-==============================
+.. _scrap_components/scrap-window:
 
-To scrap components during the manufacturing process, begin by navigating to
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+Open the Scrap pop-up window
+============================
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:37]:
```suggestion
Scrap pop-up window
===================
```

Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:37]: just a bit more concise

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 10]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
...
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window.
+The pop-up window can be accessed from an |MO| in the backend, or the Shop Floor module.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:40]:
```suggestion
The :guilabel:`Scrap` pop-up window can be accessed from an |MO| in the backend, or via the *Shop
Floor* module.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 11]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
...
+Operations --> Manufacturing Orders`. Select an open |MO|, and then click the :guilabel:`Produce
+All` button to close it.
 
-.. example::
-   If the manufacturing of a table requires four units of a table leg, and two units of the table
-   leg were scrapped during the manufacturing process, the total quantity of table legs consumed
-   will be six: four units used to manufacture the table plus two units scrapped.
+To select an |MO| that has already been closed, navigate to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, remove the :guilabel:`To Do` filter from the
+:guilabel:`Search...` bar, and then select the desired |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:58]: same 'search bar' comment as above... unsure what the right call is here

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 12]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
...
+To scrap a component from the *Shop Floor* module, begin by navigating to :menuselection:`Shop
+Floor`. Then, either click the :guilabel:`⋮ (three vertical dots)` button on an |MO| card, or select
+a work center from the top navigation, and click the :guilabel:`⋮ (three vertical dots)` button on a
+work order card.
 
-With tablet view open, click the :guilabel:`☰ (menu)` button at the top left of the screen, then
-select the :guilabel:`Scrap` button on the :guilabel:`Menu` pop-up window. The :guilabel:`Scrap`
-pop-up window then appears.
+Either method opens the :guilabel:`What do you want to do?` pop-up window. Click the
+:guilabel:`Scrap` button on the window to open the :guilabel:`Scrap` pop-up window.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:72]:
```suggestion
:guilabel:`Scrap` button on the :guilabel:`What do you want to do?` pop-up window to open the
:guilabel:`Scrap` pop-up window.
```
added specificity

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 13]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
...
-   :alt: The Scrap button on the Menu pop-up window of the manufacturing tablet view.
+Scrap pop-up window
+===================
 
-Finally, select a component from the :guilabel:`Product` drop-down menu and enter the quantity being
-scrapped in the :guilabel:`Quantity` field. Click :guilabel:`Done` to scrap the component.
+After opening the scrap pop-up window using one of the methods :ref:`detailed above
+<scrap_components/scrap-window>`, select the component or finished product being scrapped, from the
+:guilabel:`Product` drop-down menu. In the :guilabel:`Quantity` field, enter the quantity being
+scrapped.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:80]:
```suggestion
After opening the :guilabel:`Scrap` pop-up window using one of the methods :ref:`detailed above
<scrap_components/scrap-window>`, select the component or finished product being scrapped, from the
:guilabel:`Product` drop-down menu.

In the :guilabel:`Quantity` field, enter the quantity being scrapped.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 14]
```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
...
+their respective drop-down menus.
 
-Odoo also allows for finished products to be scrapped from a manufacturing order once the order is
-completed. After clicking :guilabel:`Mark as Done`, click the :guilabel:`Scrap` button to make the
-:guilabel:`Scrap` pop-up window appear.
+Enable the :guilabel:`Replenish Scrapped Quantities` checkbox if a picking order should be created
+to replace the scrapped component(s) upon confirmation of the scrap order. This option should only
+be enabled for warehouses with :doc:`two-step <two_step_manufacturing>` or :doc:`three-step
+<three_step_manufacturing>` manufacturing enabled, since components are not picked as part of the
+:doc:`one-step <one_step_manufacturing>` manufacturing process.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:91]:
```suggestion
Enable the :guilabel:`Replenish Scrapped Quantities` checkbox if a picking order should be created
to replace the scrapped component(s) upon confirmation of the scrap order.

This option should only be enabled for warehouses with :doc:`two-step <two_step_manufacturing>` or
:doc:`three-step <three_step_manufacturing>` manufacturing enabled, since components are not picked
as part of the :doc:`one-step <one_step_manufacturing>` manufacturing process.
```

**Review by ksc-odoo — APPROVED** [review]
Alrighty, @hojo-odoo --- another awesome MRP doc. Way to go. Just finished my 2nd Round Peer Review on this, and after you address the handful of comments I left, I believe this one will be ready for the next stage 👍

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 14 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:91]: I'd like to keep this paragraph together since there are only two sentences, and the second is directly referencing the first.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 15]
```diff
@@ -8,10 +8,10 @@ Terminology
 Warehouse
 ---------
 
-In Odoo, a **Warehouse** is the actual building/place in which a company's items are stocked.
-Multiple warehouses can be set up in Odoo and the user can create moves between warehouses.
+In Odoo, a *warehouse* is the actual building/place in which a company's items are stocked. Multiple
+warehouses can be set up in Odoo and the user can create moves between warehouses.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:12]:
```suggestion
warehouses can be set up in Odoo, and the user can create moves between warehouses.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 8 above)
Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:31]: Like we talked about this week, and like you confirmed with ZST, I think it would be best to go with a GUI label for the search -- but I *believe* we landed on removing the following "..." from it. If so, I think it would be best to remove them from this line.

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 16]
```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
...
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+Scrap pop-up window
+===================
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window.
+The pop-up window can be accessed from an |MO| in the backend, or the Shop Floor module.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:40]:
```suggestion
The pop-up window can be accessed from an |MO| in the backend, or the *Shop Floor* module.
```
just adding the necessary * to the module name here. Like we discussed, you don't have to add the 'Scrap' title to this sentence re: pop-up window, as it was just mentioned the sentence prior -- as you pointed out yesterday 👍

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 17]
```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
...
+To scrap a finished product from an |MO|, begin by navigating to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`. Select an open |MO|, and then click the :guilabel:`Produce
+All` button to close it.
 
-After scrapping a component, continue the manufacturing process using the required quantity of the
-component that was scrapped. The on-hand stock count for the component that was scrapped updates to
-reflect both the scrapped quantity and the quantity consumed during manufacturing.
+To select an |MO| that has already been closed, navigate to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, remove the :guilabel:`To Do` filter from the
+:guilabel:`Search...` bar, and then select the desired |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:58]: same search bar comment as above... let's go with :guilabel:`Search` bar, per the decision you got from ZST the other day (just removing the "...")

**Review by ksc-odoo — APPROVED** [review]
hey @hojo-odoo -- fantastic job on this PR. After you make these last, few, super-duper minor adjustments, I think you can go ahead and tag this for Tech Review. I'm approving now, so you can get the ball rolling on this one, once you've made these final modifications. Great work! 👍

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 8 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:31]: ZST didn't say anything about removing the "...", just that we can keep the GUI label. I'm going to keep it as is, since that's how it appears in my other docs.

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 17 above)
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:58]: Same comment as above.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst [hunk 18]
```diff
@@ -8,10 +8,10 @@ Terminology
...
 
-.. _inventory/management/difference-warehouse-location:
+.. _inventory/warehouses_storage/inventory_management/warehouses-locations/locations:
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst:14]: Hi @hojo-odoo, I'm not sure why this custom anchor was changed? 
The existing anchor `inventory/management/difference-warehouse-location` path is still valid and follows agreed upon format of `app-module/parent-folder(optional)/anchor-text`. The change you are making does not follow this format. Is there a reason this change is necessary, or can we leave the anchor as-is? LMK Thanks! 😊
```suggestion
.. _inventory/management/difference-warehouse-location:
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 19]
```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
    :guilabel:`Virtual Locations/Scrap` location.
 
-Scrap manufacturing components
-==============================
+.. _scrap_components/scrap-window:
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:34]: Custom anchor formatting
```suggestion
.. _manufacturing/management/scrap-window:
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 20]
```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
...
-pop-up window then appears.
+Scrap pop-up window
+===================
 
-.. image:: scrap_manufacturing/tablet-scrap-button.png
-   :align: center
-   :alt: The Scrap button on the Menu pop-up window of the manufacturing tablet view.
+After opening the scrap pop-up window using one of the methods :ref:`detailed above
+<scrap_components/scrap-window>`, select the component or finished product being scrapped, from the
+:guilabel:`Product` drop-down menu.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:79]:
```suggestion
<manufacturing/management/scrap-window>`, select the component or finished product being scrapped,
from the :guilabel:`Product` drop-down menu.
```

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst [hunk 21]
```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
...
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+Scrap pop-up window
+===================
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window.
+The pop-up window can be accessed from an |MO| in the backend, or the *Shop Floor* module.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:40]: Early line break
```suggestion
Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window. The
pop-up window can be accessed from an |MO| in the backend, or the *Shop Floor* module.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @hojo-odoo! Nice PR! I have a few corrections and a question about the custom anchor change, please see below. 

Lets discuss and tag me for another look when this PR is ready, thank you!

**Review by samueljlieber — APPROVED** [review]
Hi @hojo-odoo, thank you for addressing the custom anchor formatting– I agree with your change! Everything else looks great too 👍

@StraubCreative this pr is good to go!

File: content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst (hunk 8 above)
Comment (StraubCreative) [reply content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst:31]: Hi there @ksc-odoo @hojo-odoo 

There are two options I think that are acceptable:
- ``` guilabel:`Search...` bar```
- search bar

I prefer the gui in most cases because we are directly referencing it and by mirroring the placeholder text as it is on the UI, we're removing any/all doubt or chances of confusion as to what we're talking about. But it's up to the writer for what they want to do because both are *technically* correct.

> but I believe we landed on removing the following "..." from it.
Don't know where this came from 😄 ``` guilabel:`Search` bar``` isn't correct, to me, because that's not how it's labeled on the UI. If we do the gui version, please include the ellipses.

cc: @jero-odoo

**Review by StraubCreative — APPROVED** [review]
@robodoo r+
